stackrox/overlays/moc/smaug/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I dont think we need all this overlay stuff here. Should be enough to have the `Kind: Application` to let argocd pick it up